### PR TITLE
update requirements

### PR DESCRIPTION
**Pull request by azazellochg**
Needs more testing, also @fonsecareyna82 knows about black gif icons issue with new Pillow

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-requests<=2.25.1 # up to date on 07-2021
-mpi4py <= 3.0.3 # up to date on 07-2021
-tkcolorpicker<=2.1.3 # up to date on 07-2021
\ No newline at end of file
+bibtexparser<=1.2.0 # up to date on 07-2022
+psutil<=5.9.1 # # up to date on 07-2022
+configparser<=5.2.0 # # up to date on 07-2022
+matplotlib==3.3.4; python_version == '3.6'
+matplotlib==3.5.2; python_version >= '3.7' # up to date on 07-2022
+numpy<=1.23.0 # up to date on 07-2022. Careful. Xmipp is compiled agains numpy!
```
Comment (pconesa) [inline requirements.txt:6]: Shouldn't we recompile xmipp against a new version of numpy?

Comment (azazellochg) [reply requirements.txt:6]: If I install the latest scikit-learn and compile devel xmipp without cuda/opencv, it does compile without errors. This is with gcc 11.3, openmpi 4.1.4. Latest numpy 1.23.0

**Comment by azazellochg** [comment]
@pconesa should we consider reverting https://github.com/scipion-em/scipion-pyworkflow/commit/3dc793edac8b854c8e5e00ec089f207d57dc6458 since tkcolorpicker is abandoned?

File: requirements.txt (hunk 1 above)
Comment (delarosatrevin) [reply requirements.txt:6]: Did you compiled it again the libraries in conda or system wide? Can you share more tips about Xmipp compilation?

Comment (azazellochg) [reply requirements.txt:6]: Hi @delarosatrevin , I only made a new conda env with updated libs. The only problem with xmipp compilation I keep encountering is that it cant detect libhdf5 from conda properly. So I had to export the path to LD_LIBRARY_PATH

**Review by pconesa — APPROVED** [review]

**Comment by azazellochg** [comment]
Pending https://github.com/I2PC/scipion-em-xmipp/pull/572

**Comment by azazellochg** [comment]
new requirements are already outdated. By the time xmipp merges that PR there will be python 3.12. Should we update requirements.txt again @pconesa ?

**Pull request merged**